docs/reference/edot-cloud-forwarder/aws.md

```diff
@@ -0,0 +1,384 @@
+| `S3LogsJsonEncodingMode` | *(Required if `EdotCloudForwarderS3LogsType` is `json`)*<br>Defines how JSON logs are structured:<br>• `body` *(default)* – Stores logs in the request body<br>• `body_with_inline_attributes` – Logs include inline attributes |
+
+:::
+
+:::{tab-item} CloudWatch
+
+For CloudWatch logs, use the following settings:
+
+| Setting            | Description |
+| ------------------- | --- |
+| `SourceCloudWatchLogGroupARN` | Amazon Resource Name (ARN) of the CloudWatch Log Group where the subscription filter will be created. This Log Group serves as the primary storage location for logs. |
+
+:::{note}
+The log group must already exist in your AWS account and region. If the ARN points to a non-existent log group, stack deployment or updates might fail.
+:::
+
+:::
+::::
+
+### Optional settings
+
+These are optional settings you can set in the CloudFormation template:
+
+| Setting            | Description |
+| ------------------- | --- |
+| `EdotCloudForwarderVersion` | Version of the EDOT Cloud Forwarder. Expected format is semantic versioning, for example `0.1.5`. Defaults to the latest available patch version. Don't change this value unless advised by Elastic Support. |
+| `EdotCloudForwarderTimeout` | Maximum execution time for the Lambda function, measured in seconds. Default value is `300` seconds. Maximum value is `900` seconds. Minimum value is `1` second. |
+| `EdotCloudForwarderMemorySize` | Set the allocated memory for the Lambda function, measured in megabytes. Default value is `1024` MB. Maximum value is `10240` MB. Minimum value is `128` MB. | 
+| `EdotCloudForwarderConcurrentExecutions` | Set the maximum number of reserved concurrent executions for the Lambda function. Default value is `50`. Make sure this value doesn't exceed your AWS account's concurrency limit. |
+
+## Deployment examples
+
+The following examples use the CloudFormation template files hosted in the [public S3 bucket](#download-templates).
+
+- Use the `--template-url` flag to reference a template hosted on S3. 
+- To always use the most recent stable templates, use the `latest` path. For example, `v0/latest`.  
+- To pin a specific version, replace `latest` with the desired version tag. For example, `v0/v0.1.5`.  
+
+Alternatively, if you have downloaded the template file, you can use the `--template-body file://<path>` option with a local template file.
+
+::::
+
+::::{tab-set}
+
+:::{tab-item} VPC Flow logs
+
+This example deploys a CloudFormation stack to collect VPC Flow Logs stored in an S3 bucket.
+
+```sh
+aws cloudformation create-stack \
+  --stack-name edot-cloud-forwarder-vpc \
+  --template-url https://s3.amazonaws.com/edot-cloud-forwarder/v0/latest/cloudformation/s3_logs-cloudformation.yaml \
```

Comment:
Can you replace the first part of the url

```
https://s3.amazonaws.com/edot-cloud-forwarder
```
with
```
https://edot-cloud-forwarder.s3.amazonaws.com
```
and in all other occasions.